build-logic-commons/gradle-plugin/src/main/kotlin/gradlebuild.code-quality.gradle.kts

```diff
@@ -117,18 +117,6 @@ extensions.findByType<SourceSetContainer>()?.all {
     }
 }
 
-// Empty source dirs produce cache misses, and are not caught by `git status`.
-// Fail if we find any.
-tasks.withType<SourceTask>().configureEach {
```

Comment:
I suppose all our source tasks now ignore empty directories? I am sure Groovy and Java compilation do, I think Kotlin compilation does as well.

Reply:
TBH, I'm not sure. Let's see.